package.json

```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.8.4",
+    "@babel/runtime-corejs3": "^7.8.4",
```

Comment:
prob best to full uninstall the package locally i.e. use `npm uninstall @babel/runtime-corejs3 --save-dev` and reinstall (as dependency) so that the change is registered in package-lock.json

Reply:
ah yes, will do

Reply:
done